docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -1,13 +1,11 @@
 [[cluster-nodes-reload-secure-settings]]
-== Nodes Reload Secure Settings
+=== Nodes reload secure settings API
+++++
```

Comment:
This API page should follow the format specified in https://github.com/elastic/docs/blob/master/shared/api-ref-ex.asciidoc (similar to the other cluster APIs).  This cleanup can occur in a subsequent PR, however.